templates/editentity/history.php

```diff
@@ -63,4 +72,16 @@
                 <?php endif; ?>
             </div>
         <?php endforeach; ?>
+    <?php  ?>
+    <div id="history_pagination_controls" style="width: 50%">
+        <br style="clear: both">
+        <?php if (isset($this->data['history_prev_offset'])): ?>
+        <a style="display: inline-block; float: left"
+           href="editentity.php?eid=<?= htmlspecialchars($entity->getEid()); ?>&selectedtab=8&history_offset=<?= $this->data['history_prev_offset'] ?>#history">&leftarrow; Later revisions</a>
```

Comment:
`htmlspecialchars` should be `urlencode`, `&` should be `&amp;` I guess...
Same below.